### ISSUE TITLE
Syntax highlighting breaks when type argument spans multiple lines

### ISSUE THREAD

**Issue by evaera** [styled-components/vscode-styled-components#277]
**Describe the bug (including copyable syntax)**

When typing passed props to a styled component with TypeScript with the type argument, syntax highlighting breaks if the type argument spans multiple lines.

```tsx
const StyledStudioBlurb = styled.section<{
  reverse: boolean
}>`
  flex-direction: ${(props) => (props.reverse ? "row-reverse" : "row")};
`
```

**Screenshot**
Broken:
![image](https://user-images.githubusercontent.com/2489210/112898796-4bc7e480-90af-11eb-8844-bddb1788ca48.png)

Works when squished to single line:
![image](https://user-images.githubusercontent.com/2489210/112898832-55e9e300-90af-11eb-8c31-aa98ed41a8e5.png)

**Expected behavior**
Syntax highlighting should still work in this case.

**Build environment (please complete the following information):**

- OS: Windows 10
- VSCode Version: 1.54
- Extension Version: 1.5.0

**Comment by jasonwilliams** [comment]
Yep it appears regex patterns won't work on multi-lines so some of the syntax definition needs a re-think

**Comment by jasonwilliams** [comment]
This has been fixed in [v1.5.1](https://github.com/styled-components/vscode-styled-components/releases/tag/v1.5.1) please let us know or raise an issue if this is still happening
